### PR TITLE
Remove dependency artifacts and update docs

### PR DESCRIPTION
**Pull request by localjo**
Just some cleanup of some outstanding items on #503

### CODE REVIEW

**Comment by localjo** [comment]
This should be ready to merge if it looks good to you guys.

**Comment by localjo** [comment]
@ZachTRice @Benjaki2 @minniewong @rboller I'd like your feedback on my changes to developing.md in this PR. If there is anything lacking in my descriptions of the Grunt tasks, let me know. The intention is to keep the descriptions brief while explaining why someone would use that task and what the results of the task will be. Here's a link to view my changes to the file: https://github.com/nasa-gibs/worldview/pull/615/files#diff-e0bc0ebf70515a5abdf1ae720b279282

**Comment by ZachTRice** [comment]
I'm not in favor of completely removing the style_guide document. Not everyone is going to have a linter to view how their code should be formatted. Without this documentation there is not mention of how to format CSS and the modifications we have made to standardjs. This is also a good place to document any other kind of styles we want to maintain consistency with such as creating elements. If someone wanted to add a UI element, eventually, it would be good to have a place to say all modals should have 3px radius border, use this set of colors, etc.

File: doc/style_guide.md [hunk 1]
```diff
@@ -1,151 +0,0 @@
-# Developer Guide
```
Comment (ZachTRice) [inline doc/style_guide.md]: This document should not be removed in my opinion. There is a lot of useful information here.

**Comment by localjo** [comment]
@Benjaki2 I'm in favor of moving `eslint` and `stylelint` out of Grunt. Should be pretty easy to do, but I can do it in another PR since it's a separate change.

File: doc/style_guide.md (hunk 1 above)
Comment (localjo) [reply doc/style_guide.md]: This moved to CONTRIBUTING.md

**Comment by localjo** [comment]
@ZachTRice I moved the style guide information here: https://github.com/nasa-gibs/worldview/pull/615/files#diff-6a3371457528722a734f3c51d9238c13R27

**Comment by localjo** [comment]
A visual style guide is a completely separate issue from a code style guide. I don't think we need to be concerned about that in this PR.

File: doc/style_guide.md (hunk 1 above)
Comment (ZachTRice) [reply doc/style_guide.md]: Seems like we are loosing a lot of important information in the process.

File: doc/developing.md [hunk 2]
```diff
@@ -1,156 +1,53 @@
...
 
-* ``dist/<name>-<version>-<release>.git<revision>.tar.bz2``
+`grunt config`* : Updates branding and configuration options.
 
-Where ``name``, ``version``, ``release`` are the values found in the
-``package.json`` file and ``revision`` is the value from:
+`grunt site`* : Generates build assets for deployment.
 
-    git rev-parse --short HEAD
+`grunt fetch`* : Makes a request to the GIBS `GetCapabilities` API and updates layer configurations.
```
[the remark this answers is not in the document]
Comment (minniewong) [reply doc/developing.md]: agreed

File: doc/style_guide.md (hunk 1 above)
Comment (localjo) [reply doc/style_guide.md]: No, this entire style guide document has been replaced and is irrelevant now.

Comment (localjo) [reply doc/style_guide.md]: The problem with trying to maintain a `.stylelintrc`/`.eslintrc` and a separate description of those rules is that there are then two sources of truth and the two will never stay in sync. It's best to have people refer to once source of truth, which is the place where the rules are configured, `.stylelintrc` and `.eslintrc`.

Comment (ZachTRice) [reply doc/style_guide.md]: True, updating two locations for one change doesn't make sense. I still feel like we are loosing something by not having real examples but if I am in the minority on this then I'm not going to hold up the PR over it.

**Pull request merged**